content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+
+In this example, an API test is created to ensure your website is constantly up and providing responses in a specific amount of time.
+
+### Configure the request
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add a start URL.
+    - Select **Advance Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add one or two tags, like `prod`, to help organize and filter tests.
+    - Select locations for testing.
+    - Click the **Test URL** button.
+
+#### Define your alert conditions
+
+Now that the test is completed, if you did not create any [assertions][9], they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.
+
+In this example, three default assertions are populated when testing the URL:
+```
+When status code is 200
+And header content-type is text/html; charset=utf
+And response time less than 2000 ms
+```
+
+These assertions define the alert condition and can be customized. To add a custom assertion, click on any response header in the response preview. You can also click the **New Assertion** button to add an assertion manually, (e.g., `body` contains `Shop.ist`.)
+
+To configure the manual retry of a test when it fails, set:
+```
+An alert is triggered if your test fails for x minutes from any x of x locations`
+Retry x time before location is marked as failed
+```
+
+Once alert conditions are set, create a message for the alert and specify what services and team members to receive the alert notification email and click **Save Test**. You can also use [integrations][10], such as Slack, Pagerduty, webhooks, etc., to receive alert notifications.
+
+{{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+### Test results
+
+An **API test** homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results. Sample results include errors, resources, and traces. 
+
+To troubleshoot a failed test, scroll to the Test Results section and click on the **Test Results** tab. Click on the failed test, labeled as `Alert`, to view detailed test results. Review the failed assertions and response details such as returned status code, response time, and associated headers and body to resolve the issue.
+
+{{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure" responsive="true" style="width:90%;">}}
```

Comment:
Let's use a screenshot with the same endpoint https://cl.ly/3780b2df1064

Reply:
Used your screenshot example.